src/liballoc/collections/linked_list.rs

```diff
@@ -1843,3 +1843,15 @@ unsafe impl<T: Send> Send for IterMut<'_, T> {}
 
 #[stable(feature = "rust1", since = "1.0.0")]
 unsafe impl<T: Sync> Sync for IterMut<'_, T> {}
+
+#[unstable(feature = "linked_list_cursors", issue = "58533")]
```

Comment:
```suggestion
#[stable(feature = "linked_list_cursors", since = "1.45.0")]
```

Reply:
But it doesn't make sense for an unstable type to have stable impls, i think?

Reply:
I'm curious too.

Reply:
It's fine to leave this as unstable, it will get stabilized along with cursors.